http2_general/test_flow_control_window.py

```diff
@@ -52,15 +67,28 @@ def test_single_stream(self):
         6. Client send WindowUpdate with 1K flow control window after receiving first DATA frame;
         7. Tempesta forward second DATA frame with 1k bytes;
         """
-        client, server = self._initiate_client_and_server(
-            response=(
+        if not trailer:
+            response_str = (
                 "HTTP/1.1 200 OK\r\n"
                 + f"Date: {HttpMessage.date_time_string()}\r\n"
                 + "Server: debian\r\n"
                 + "Content-Length: 2000\r\n\r\n"
                 + ("x" * 2000)
             )
-        )
+        else:
+            response_str = (
```

Comment:
Please use response as test param. You should not use if/else construction if you can use test params. This makes tests easier.